Documentation/DataTypes/_FunctionName.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace MyVendor\MyExtension\SomeNamespace;
```

Comment:
Actually, I find the "SomeNamespace" strange as the whole string "MyVendor\MyExtension\SomeNamespace" is the namespace. In other examples we use just "MyVendor\MyExtension" as namespace.

Just my 2 cents after merging, again.

Reply:
As over 90% of PHP files stays in sub-directories of `Classes` I prefer the 3-parts-namespace. But yes, I don't like "SomeNamespace". Maybe call it `MyService` or whatever. Having classes directly in `Classes` is very very rare.